lib/features/onboarding.dart/view/screen/onboarding_screen.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:flutter/material.dart';
+import 'package:galaxyxgate/core/helpers/navigation_extention.dart';
+import 'package:galaxyxgate/core/routes/routes.dart';
+import 'package:galaxyxgate/core/themes/app_colors.dart';
+import 'package:galaxyxgate/core/themes/app_images.dart';
+import 'package:galaxyxgate/features/onboarding.dart/data/models/content.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:galaxyxgate/features/onboarding.dart/view/widget/animated_page_indicator.dart';
+import 'package:galaxyxgate/features/onboarding.dart/view/widget/onboarding_item.dart';
+import 'package:galaxyxgate/features/onboarding.dart/view/widget/positioned_star_with_animation.dart';
+
+class OnBoardingScreen extends StatefulWidget {
+  const OnBoardingScreen({super.key});
+
+  @override
+  State<OnBoardingScreen> createState() => _OnBoardingScreenState();
+}
+
+class _OnBoardingScreenState extends State<OnBoardingScreen>
+    with SingleTickerProviderStateMixin {
+  int currentIndex = 0;
+  PageController? _controller;
+  late AnimationController _animationController;
+  late Animation<double> _animation1, _animation2, _animation3;
```

Comment:
_animation1, _animation2, _animation3 ... that's not good naming, think of better descriptive names for these animation instances please :)